===================================================================
RCS file: /home/kde/kdesdk/kbabel/common/Makefile.am,v
retrieving revision 1.23
retrieving revision 1.24
diff -u -r1.23 -r1.24
--- kdesdk/kbabel/common/Makefile.am	2002/09/08 14:47:37	1.23
+++ kdesdk/kbabel/common/Makefile.am	2002/09/21 10:51:13	1.24
@@ -29,6 +29,8 @@
 # let automoc handle all of the meta source files (moc)
 METASOURCES = AUTO
 
+KDE_OPTIONS = nofinal
+
 version.h: $(top_srcdir)/kbabel/VERSION
 	printf "#undef VERSION\n#define VERSION \"`cat $(top_srcdir)/kbabel/VERSION`\"\n" > version.h
 
